data/modular_1/data_setup_additional/info.txt

```diff
@@ -0,0 +1,3 @@
+Genotype : WT
```

Comment:
There are spaces around : in the 1st line not in the next ones. Is this intentional?

Reply:
Yes